config.json

```diff
@@ -1043,6 +1389,16 @@
         "prerequisites": [],
         "difficulty": 3,
         "status": "deprecated"
+      },
+      {
+        "slug": "list-ops",
+        "name": "List Ops",
+        "uuid": "b4a24540-f4d8-4520-a1a0-14144ab68dbe",
+        "practices": [
+          "procs-blocks"
+        ],
+        "prerequisites": [],
+        "difficulty": 5
```

Comment:
Based on the difficulty ratings for other exercises on this track, I think a 3 or 4 would be more appropriate, especially since the provided skeleton includes most of what's needed to implement generics.